synapse/lib/aha.py

```diff
@@ -368,3 +535,186 @@ async def signUserCsr(self, csrtext, signas=None):
         pkey, cert = self.certdir.signUserCsr(xcsr, signas=signas)
 
         return self.certdir._certToByts(cert).decode()
+
+    def _getAhaUrls(self):
+        urls = self.conf.get('aha:urls')
+        if urls is not None:
+            return urls
+
+        ahaname = self.conf.get('aha:name')
+        if ahaname is None:
+            return None
+
+        ahanetw = self.conf.get('aha:network')
+        if ahanetw is None:
+            return None
+
+        # TODO this could eventually enumerate others via itself
+        return f'ssl://{ahaname}.{ahanetw}'
```

Comment:
we should probably raise in this case. ssl:// isn't guaranteed by designed.

Reply:
This allows it to "just work" using the provided instructions / sane defaults. They could always over-ride in the cell.yaml. I'd prefer to keep this if we can.